lib/course_planner/attendances/attendances.ex

```diff
@@ -27,10 +27,12 @@ defmodule CoursePlanner.Attendances do
       join: s in assoc(oc, :students),
       join: c in assoc(oc, :classes),
       join: a in assoc(c,  :attendances),
+      join: as in assoc(a, :student),
```

Comment:
I don't get why these two new joins are necessary

Reply:
we have had this problem with "get_course_attendances" and fixed it with this joins and sort them
in the show and fill I use offered_course.students to create a part of the table but then iterate throw offered_course.class.attendance ( which has student link too ) to check the attendance of each student, so both should be sorted the same in order columns and attendance data to match